README.md

```diff
@@ -1,3 +1,77 @@
 # sot-gepetto-viewer
 
 Viewing tools for Stack of Tasks library.
+
+This document will show the steps to install sot-gepetto-viewer and configure gepetto-gui.
+
+Important : This plugin will only work if an instance of gazebo can work on your computer.
+
+Before doing anything, you will need to install several packages that are needed for the installation process, you will need to install the following packages:
+ - sudo apt install robotpkg-qt4-qgv
+ - sudo apt install robotpkg-py27-qt4-gepetto-pythonqt		(make sure your version of python is the 2.7)
+ - sudo apt-get install libqtwebkit4
+ - sudo apt install cmake-curses-gui				(will allow you to use the ccmake command)
+ - sudo apt-get install libomniorb4-dev
+ - sudo apt-get install libopenscenegraph-dev
+ - sudo apt-get  install omniidl-python
+
+
+Now you will need to get 3 git repository on your computer, you will need to install them as follow.
+You will need to install gepetto-viewer, gepetto-viewer-corba and sot-gepetto-viewer.
+
+Installing gepetto-viewer:
+ 1. clone the repository on your computer : git clone --recursive https://github.com/Gepetto/gepetto-viewer 		(--recursive will allow you to use the cmake command)
+ 2. create of folder name _build in the folder gepetto-viewer
+ 3. open a terminal in the new folder
+  1. in the terminal, do "cmake .."
+  2. do "ccmake .."
+   1. make sure to change the variable BUILD_PY_QCUSTOM_PLOT, BUILD_PY_QGV and GEPETTO_GUI_HAS_PYTHONQT to ON
+   2. change the CMAKE_INSTALL_PREFIX path to a new folder Install (this path will be important later)
+  3. do "make install"
```

Comment:
Isn't it better to use gepetto-viewer from robotpkg in those instructions ? They would be a lot shorter.

Reply:
For the moment, gepetto-viewer from robotpkg is not usable, the variables BUILD_PY_QCUSTOM_PLOT & BUILD_PY_QGV, that are needed, are not activated.
So for the moment this is the only installation method.